packages/components/src/components/tooltip/tooltip--icon.hbs

```diff
@@ -4,7 +4,81 @@
   This source code is licensed under the Apache-2.0 license found in the
   LICENSE file in the root directory of this source tree.
 -->
+<strong>WCAG 2.1 compliant</strong>
+<p>start</p>
+<br>
+{{!-- @todo Had to add a modifier to the parent to correct the animation but in the next major release it could be removed in favor of the new HTML format --}}
+<button
+  class="{{@root.prefix}}--tooltip__trigger {{@root.prefix}}--tooltip--a11y {{@root.prefix}}--tooltip__trigger--icon {{@root.prefix}}--tooltip--left {{@root.prefix}}--tooltip--align-start">
+  {{!-- @todo Open to suggestions for this element's className; In WH we have a utility class (wh-u-a11yTitle) that can be applied to other components --}}
```

Comment:
seems out-of-scope for this pr, but I'd vote for a global class for these a11y styles as they could be very useful in other situations. Then, `{{@root.prefix}}--tooltip__content` could bring those in for this use-case, but not have them locked to this component.